coffeecard/CoffeeCard.Models/Entities/MenuItem.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+
+namespace CoffeeCard.Models.Entities;
+
+/// <summary>
+/// An item or good that Analog offers, such as a cappuccino or a t-shirt.
+/// </summary>
+public class MenuItem
+{
+    /// <summary>
+    /// ID
+    /// </summary>
+    public int Id { get; set; }
+
+    /// <summary>
+    /// The name of this menu item
+    /// </summary>
+    public string Name { get; set; }
+
+    /// <summary>
+    /// The product(s) that are eligible to redeem this menu item
+    /// </summary>
+    public ICollection<Product> Products { get; set; }
```

Comment:
Could the property be named like AssociatedProducts or ProductEligbilities? I think this express the relationship more clearly :)

Reply:
Addressed in fe39ffed2d7d416aee95f353586cf80e70afeb6f